### DISCUSSION TITLE
How to get some of the new achievements?

### DISCUSSION THREAD

**Discussion by ghost** [Schweinepriester/github-profile-achievements#12] [Q&A]
YOLO - tried to push to a private repo, but did not achieve it when I checked this morning still.
Pull Shark - Have merged at least 2 PRs in a private repo (forked from friend), but did not achieve it.
Galaxy brain - Do I have to be the one to create two discussions? and mark them as answers?

Seems a bit unclear and random on how to achieve the new achievements.

Please help if anyone has a clue...

**Comment by ale-lit** [comment]
More info on this links:

YOLO - https://github.com/Schweinepriester/github-profile-achievements/discussions/6
Pull Shark - May be repo shouldn't be private? Try it (How create PR: https://youtube.com/watch?v=rgbCcBNZcdQ)
Galaxy brain - No. You must not be the author of discussions. Tests: https://github.com/Schweinepriester/github-profile-achievements/discussions/9 & https://github.com/Schweinepriester/github-profile-achievements/discussions/7

**Reply by ghost** [reply]
Thanks will try

**Comment by kirollosatef** [comment]
Maybe this can help https://github.com/Schweinepriester/github-profile-achievements
